packages/chrome/src/elements/Chrome.tsx

```diff
@@ -38,6 +40,27 @@ const Chrome = React.forwardRef<HTMLDivElement, IChromeProps & ThemeProps<Defaul
     const isLight = hue ? isLightMemoized : false;
     const isDark = hue ? !isLightMemoized : false;
     const chromeContextValue = { hue: hue || 'chromeHue', isLight, isDark };
+    const environment = useDocument(theme);
+
+    useEffect(() => {
+      if (!environment) {
+        return undefined;
+      }
+
+      const htmlElement = environment.querySelector('html');
+
+      if (!htmlElement || isFluid) return undefined;
+
+      const defaultHtmlPosition = htmlElement.style.position;
+
+      if (!isFluid) {
+        htmlElement.style.position = 'fixed';
+      }
+
+      return () => {
+        htmlElement.style.position = defaultHtmlPosition;
+      };
+    }, [environment, isFluid]);
```

Comment:
Falsy logic tends to be difficult to parse and is missing the opportunity to short-circuit. Alternative:

```js
if (environment && !isFluid) {
  const htmlElement = environment.querySelector('html');

  if (htmlElement) {
    const defaultHtmlPosition = htmlElement.style.position;

    htmlElement.style.position = defaultHtmlPosition;

    return () => {
      htmlElement.style.position = defaultHtmlPosition;
    }
  }
}
```

Reply:
Thanks that's easier to read.